iotilecore/test/test_hw/test_virtualadapter.py

```diff
@@ -156,3 +167,11 @@ def test_realtime_tracing(tracer_hw):
 
     wrong_data = [x for x in words if (x != 'hello' and x != 'goodbye' and x != '')]
     assert len(wrong_data) == 0
+
+def test_virtual_scan(realtime_scan_hw):
+    """Make sure we can scan for virtual devices and connect directly without connect_direct
+    """
+    devices = report_hw.scan()
```

Comment:
Note that the continuous integration tests failed on this line as well:

https://travis-ci.org/iotile/coretools/builds/234175957?utm_source=github_status&utm_medium=notification